docs/walk-through/README.md

```diff
@@ -0,0 +1,114 @@
+# "Step-by-step Walk-Through" or 
+# How to Use a Hardware Wallet with Bitcoin Core Wallet
+
+Summary: On this page we describe step-by-step and show in screenshots how to use a hardware wallet and HWI 
+together with a Bitcoin Core Wallet. As hardware wallet example we have used a Trezor.
+
+## Create a watch-only Bitcoin Core wallet for Trezor
+Create your watch-only Bitcoin Core Wallet as well described in [Using Bitcoin Core with Hardware Wallets](../bitcoin-core-usage.md). 
+You find all the details in this link. But in summary, one opens one terminal and runs `bitdoind`. E.g.
+
+```
+bitcoind -testnet  -datadir=$HOME/.bitcoin-testnet -rpcallowip=127.0.0.1 -rpcuser=test -rpcpassword=somedifficultpassword
+```
+for a testnet `bitcoind` daemon, or
+```
+bitcoind -rpcallowip=127.0.0.1 -rpcuser=test -rpcpassword=somedifficultpassword
+```
+for a mainnet, i.e. regular, `bitcoind` daemon.
+
+Then in another terminal run commands similar to these, adapted to your environment:
+```
+hwi.py enumerate # this shows you the fingerprint of hardware wallet
+TREZORFINGERPRINT_TESTNET="yourTrezorFingerprint" # shown by "hwi enumerate"
+# in this example we use SEGWIT BECH32 ADDRESSES
+DERIVATIONPATH_TESTNET=1 # testnet uses derivation paths like m/84h/1h/0h/0/* and m/84h/1h/0h/1/* on testnet
+DERIVATIONPATH_MAINNET=0 # mainnet uses derivation paths like m/84h/0h/0h/0/* and m/84h/0h/0h/1/* on testnet
+# if the mainnet path is used on testnet, it will work too but Trezor device gives warnings 
+# of unknown address on Trezor display
+wallet=wallet.test
+rec=$(hwi --testnet -f $TREZORFINGERPRINT_TESTNET getkeypool --wpkh --path m/84h/${DERIVATIONPATH_TESTNET}h/0h/0/* --keypool 0 1000)
+chg=$(hwi --testnet -f $TREZORFINGERPRINT_TESTNET getkeypool --wpkh --path m/84h/${DERIVATIONPATH_TESTNET}h/0h/1/* --keypool --internal 0 1000)
```

Comment:
`--wpkh` was replaced with `--addr-type wpkh`.

Additionally, it is unnecessary to specify `--path` if you are planning on just using the default path. To get the testnet path, you can use the `--testnet` option.

Lastly, without `--path`, `getkeypool` will return both change and receiving descriptors, so it is unnecessary to have call it twice.

Reply:
Ok, Andrew, I replaced the `--wpkh` with `--addr-type wpkh` so that this option is up-to-date.

I know and I understand that by removing `--path` that with a single command both receiving and change address will be handled at once and that there are fewer calls, i.e. the script is shorter. However, I copied this code out of my script that I used for testing many different scenarios. I find it definitely helpful, for users to see which address they are creating, so the next time around e.g. they can do address 1000 through 2000, etc. I personally also found it useful to see path so that the script can be adapted to different networks. In my humble opinion it helps the user understand better and through these examples the user learns more and can adapt this script to more situations. 

In short, I was not going for the shortest code, or the simplest solution, but for the solution that teaches the most and demos the most info. As such, I have left the options `--path` and the additional `--keypool` arguments in place. 

What do you think @achow101 ? Reasonable?